x/mongo/driver/operation.go

```diff
@@ -855,18 +856,18 @@ func (op Operation) retryable(desc description.Server) bool {
 
 // roundTrip writes a wiremessage to the connection and then reads a wiremessage. The wm parameter
 // is reused when reading the wiremessage.
-func (op Operation) roundTrip(ctx context.Context, conn Connection, wm []byte) (result, pooledSlice []byte, err error) {
+func (op Operation) roundTrip(ctx context.Context, conn Connection, wm []byte) (result []byte, err error) {
```

Comment:
What are your thoughts on the named return parameters? A lot of these functions are small and the documentation on them is fairly clear. IMO we would remove them.

Reply:
I agree, at least for this one. `result` is not even used.